src/components/control-bar/ControlBar.svelte

```diff
@@ -4,15 +4,7 @@
   SPDX-License-Identifier: MIT
  -->
 
-<style>
-  div {
-    background-color: #00a000;
-    position: sticky;
-    top: 0;
-    z-index: 4;
-  }
-</style>
-
-<div class="min-h-16 px-10 flex w-full items-center justify-between">
+<nav
```

Comment:
I think this should be a div. I'd say the TabView component is the nav.

Reply:
Good point, the TabView is a `nav`

Maybe both are `nav`s since the top contains navigation to home page and other external pages, and I just add an `aria-label` to them to distinguish the difference between the navigations? 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#labeling_section_content

Reply:
I wouldn't include the language menu, cookies or about dialog opener as nav items though, so it's only really the home page link that's a single nav item. One to bring for discussion tomorrow?

Reply:
As discussed, not going to semantically label the top bar as `navigation`